client/src/components/base/ArrowButton/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { baemin } from '@/static/style/common';
+
+const ArrowButton = () => {
+  const [isActive, setIsActive] = useState(false);
+
+  const standardHeight = 800;
+
+  document.addEventListener('scroll', () => {
+    if (isActive && window.scrollY < standardHeight) setIsActive(false);
+    else if (!isActive && window.scrollY > standardHeight) setIsActive(true);
+  });
+
+  const onClickHandler = (e: React.MouseEvent) => {
+    const { target } = e;
+    if (!(target instanceof HTMLElement)) return;
+
+    if (target.closest('.arrow-button')) {
+      const scrollTo = document.querySelector('.header');
+      scrollTo.scrollIntoView({ behavior: 'smooth' });
+    }
```

Comment:
```suggestion
    if (target.closest('.arrow-button')) {
      window.scrollTo({top: 0,left: 0, behavior: 'smooth'})
    }
```
나중에 header가 변경될 수도 있다면 그냥 `winodw top 0`으로 이동시키는건 어떤가요?

Reply:
scrollTo({top: 0,left: 0, behavior: 'smooth'})
 behavior: 'smooth이거 때문에  scrollIntoView 이거 쓴건데 되네요?!
정말 GOOD입니다!